src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -411,6 +411,43 @@ CHIP_ERROR TestAttrAccess::WriteStructAttribute(AttributeValueDecoder & aDecoder
 bool emberAfTestClusterClusterTestCallback(app::CommandHandler *, const app::ConcreteCommandPath & commandPath,
                                            const Test::DecodableType & commandData)
 {
+    // Setup the test variables
+    emAfLoadAttributeDefaults(commandPath.mEndpointId, true, MakeOptional(commandPath.mClusterId));
+    for (int i = 0; i < kAttributeListLength; ++i)
+    {
+        gListUint8Data[i] = 0;
+        gListOctetStringData[i].SetLength(0);
+        gListOperationalCert[i].SetLength(0);
+        listStructOctetStringData[i].fabricIndex     = 0;
+        listStructOctetStringData[i].operationalCert = ByteSpan();
+        gSimpleEnums[i]                              = SimpleEnum::kUnspecified;
+    }
+    gSimpleEnumCount = 0;
+
+    gStructAttributeValue.a = 0;
```

Comment:
Argh...we shouldn't need to do this given support for initialization to defaults went in recently, but for some reason, some of the structs got missed in that I think.

@bzbarsky-apple FYI

Reply:
i.e the ideal prescription here would be to just do:

```
gStructAttributeValue = Structs::SimpleStruct::Type()
```

Reply:
I think this would be handled by a separate PR from Boris. Lets not hold up this PR on a dependency.